ledger-api/grpc-definitions/com/daml/ledger/api/v1/transaction_service.proto

```diff
@@ -140,3 +143,16 @@ message GetLedgerEndResponse {
   LedgerOffset offset = 1;
 }
 
+message GetLatestPrunedOffsetsRequest {
+  // Empty for now, but may contain fields in the future.
+}
+
+message GetLatestPrunedOffsetsResponse {
+  // The offset up until the ledger has been pruned, without pruning divulgence events.
```

Comment:
I think divulged events can still be pruned? The think that differentiates this is offset from the other one is whether or not divulgence events were pruned indiscriminately. 
```suggestion
  // The offset up until which the ledger has been pruned.
  // This corresponds to a pruning mode where divulgence events are pruned only if there is a corresponding archive event within the pruning offset.
```

Reply:
Is it accurate to say that the *ledger* has been pruned?
What about this saying ~ *data indexed from the ledger by this participant server*?

Reply:
It is actually all the ledger views, not just what was indexed by the participant server.

Reply:
> The think that differentiates this is offset from the other one is whether or not divulgence events were pruned indiscriminately.

You are right that divulgence events with associated archivals present would still be pruned even in normal pruning. Will clarify to avoid confusion.